Sources/ACKategories/SwiftUIExtensions/View+FrameSize.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+@available(iOS 13, *)
+extension View {
+
+    public func frame(size: CGSize) -> some View {
+        assert(size.width.isFinite)
+        assert(size.width >= 0)
+        assert(size.height.isFinite)
+        assert(size.height >= 0)
+        return frame(width: size.width, height: size.height)
+    }
+
+    public func frame(size: CGSize, alignment: Alignment) -> some View {
+        frame(width: size.width, height: size.height, alignment: alignment)
```

Comment:
Also we have some asserts in the variant without alignment, so we should rather be consistent and have in both variants or omit it.